includes/Modules/AdSense.php

```diff
@@ -195,17 +206,18 @@ public function get_debug_fields() {
 	 */
 	protected function get_datapoint_definitions() {
 		return array(
-			'GET:adunits'       => array( 'service' => 'adsense' ),
-			'GET:accounts'      => array( 'service' => 'adsense' ),
-			'GET:alerts'        => array( 'service' => 'adsense' ),
-			'GET:clients'       => array( 'service' => 'adsense' ),
-			'GET:report'        => array(
+			'GET:adunits'                  => array( 'service' => 'adsense' ),
+			'GET:accounts'                 => array( 'service' => 'adsense' ),
+			'GET:alerts'                   => array( 'service' => 'adsense' ),
+			'GET:clients'                  => array( 'service' => 'adsense' ),
+			'GET:report'                   => array(
 				'service'   => 'adsense',
 				'shareable' => Feature_Flags::enabled( 'dashboardSharing' ),
 			),
-			'GET:notifications' => array( 'service' => '' ),
-			'GET:urlchannels'   => array( 'service' => 'adsense' ),
-			'GET:sites'         => array( 'service' => 'adsense' ),
+			'GET:notifications'            => array( 'service' => '' ),
+			'GET:urlchannels'              => array( 'service' => 'adsense' ),
+			'GET:sites'                    => array( 'service' => 'adsense' ),
+			'GET:ad-blocking-recovery-tag' => array( 'service' => 'adsense' ),
```

Comment:
Please sort. The convention is to sort by datapoint first, then method. This is more relevant for the switch below because it's so long but the two should match.